kano/utils/audio.py

```diff
@@ -62,26 +69,15 @@ def get_volume():
     from kano.logging import logger
 
     percent = 100
-    millibel = 400
 
-    cmd = "amixer | grep %"
-    o, _, _ = run_cmd(cmd)
-    o = o.strip().split(' ')
-
-    try:
-        millibel = int(o[2])
-    except Exception:
-        msg = 'asmixer format bad for millibel, o: {}'.format(o)
-        logger.error(msg)
-        pass
+    cmd = "amixer | head -n 6 | grep -Po '(\d{1,3})(?=%)'"
+    output, _, _ = run_cmd(cmd)
 
     try:
-        percent = int(o[3].translate(None, '[]%'))
+        percent = int(output)
     except Exception:
-        msg = 'asmixer format bad for percent, o: {}'.format(o)
+        msg = 'amixer format bad for percent, output: {}'.format(output)
         logger.error(msg)
         pass
 
-    # logger.debug('percent: {}, millibel: {}'.format(percent, millibel))
-
-    return percent, millibel
+    return percent
```

Comment:
There is a different helper fn that allows translation between the two, for the apps that need it https://github.com/KanoComputing/kano-toolset/blob/jessie/kano/utils/audio.py#L36.

If you go ahead with this interface change though, you should probably bump this package's version and then depend in the packages that are affected